### PR TITLE
Make a small adjustment to `:source` command

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-operations-1084.surge.sh

File: modules/ROOT/pages/tools/cypher-shell.adoc [hunk 1]
```diff
@@ -262,7 +262,7 @@ You can use the `:source` command followed by the file name to run the Cypher st
 
 [source, shell]
 ----
-:source example.cypher
+:source /tmp/example.cypher
```
Comment (renetapopova) [inline modules/ROOT/pages/tools/cypher-shell.adoc:265]: I would suggest that we put something more general, as I am not sure if everyone puts their Cypher scripts in the /tmp folder.
```suggestion
:source /path/to/your/example.cypher
```

**Review by renetapopova — COMMENTED** [review]
I added a small suggestions.

File: modules/ROOT/pages/tools/cypher-shell.adoc (hunk 1 above)
Comment (NataliaIvakina) [reply modules/ROOT/pages/tools/cypher-shell.adoc:265]: Thank you! It's a good idea.

**Review by renetapopova — APPROVED** [review]

**Pull request merged**